go/dpagg/aggregationState.go

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package dpagg
+
+type aggregationState int
+
+var errorMessages = []string{"", " object has been already merged.", " object has been already serialized.", "Noised result is already computed and returned."}
+var stateName = []string{"Default", "Merged", "Serialized", "ResultReturned"}
+
+const (
```

Comment:
As agreed change to 
const (
	Default = iota
	Merged
	Serialized
	ResultReturned
)

var errorMessages = map[int]string {
	Default: "",
	Merged: "object has been already merged",
	Serialized: "object has been already serialized",
	ResultReturned: "noised result is already computed and returned",
}

so this is not that fragil